### DISCUSSION TITLE
Is there a way to add a details or descriptions field to image files?

### DISCUSSION THREAD

**Discussion by owltree87** [stashapp/stash#4045] [Q&A] [answered]
Pretty much just the title. Is there a way to add a details or descriptions field to image files similar to the details field you have in video files? Is there a way to enable one in settings, any custom code you can inject to enable one or is this a planned feature in future? Thanks

**Comment by dingdongsolong4** [comment, marked as answer]
There is indeed a feature request for this, currently on the Backlog milestone: #1443.
